RSSReader-Maruyama/DataManager/UserDataManager.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  LoginChecker.swift
+//  RSSReader-Maruyama
+//
+//  Created by 丸山翔 on 2020/09/03.
+//  Copyright © 2020 丸山翔. All rights reserved.
+//
+
+import UIKit
+
+class UserDataManager: NSObject {
+    
+    private let UD_USER_DATA_KEY = "userdata"
+    lazy var userDefault = UserDefaults.standard
+
+    
+    func isLoggedIn() -> Bool {
+        let user = userDefault.object(forKey: UD_USER_DATA_KEY ) as? Data
+        
+        if user != nil {
+            return true
+        } else {
+            return false
+        }
+    }
+    
+    func getUserData() -> User {
+        let userData = self.userDefault.object(forKey: UD_USER_DATA_KEY) as! Data
```

Comment:
```suggestion
        guard let userData = userDefaults.data(forKey: USER_DATA_KEY) else {
            return nil
        }
```
selfは基本的に付けなくて大丈夫です。
userDataが取得できるとは限らないので、guard文を使用してnilだった場合は即時returnするようにしましょう。

Reply:
修正しました。